docs/source/guide/explanation/additional_features/tiling.rst

```diff
@@ -0,0 +1,114 @@
+    | INSTANCE_SEGMENTATION |    Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50     |    MaskRCNN-ResNet50     |     otx/algorithms/detection/configs/instance_segmentation/resnet50_maskrcnn/template.yaml    |
+    | INSTANCE_SEGMENTATION | Custom_Counting_Instance_Segmentation_MaskRCNN_EfficientNetB2B | MaskRCNN-EfficientNetB2B | otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/template.yaml |
+    +-----------------------+----------------------------------------------------------------+--------------------------+-----------------------------------------------------------------------------------------------+
+
+To enable tiling in OTX training, set the tiling_parameters.enable_tiling parameter to 1. Here's an example of enabling tiling for the SSD model template:
+
+.. code-block::
+
+    otx train Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects params --tiling_parameters.enable_tiling 1
+
+.. note::
+
+    To learn how to deploy the trained model and run the exported demo, refer to :doc:`../../tutorials/base/deploy`.
+
+    To learn how to run the demo in CLI and visualize results, refer to :doc:`../../tutorials/base/demo`.
+
+
+Customizing Tiling Parameters
+*****************************
+
+Users have the flexibility to customize the tiling process by disabling adaptive tiling parameters and setting the following parameters:
+
+.. code-block:: 
+    
+    otx train Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects
+    params --tiling_parameters.enable_tiling 1          \  # enable tiling
+           --tiling_parameters.enable_adaptive_params 0 \  # disable automatic tiling parameter optimization
+           --tiling_parameters.tile_size 512 \             # tile size configured to 512x512
+           --tiling_parameters.tile_overlap 0.1 \          # 10% overlap between tiles
+
+By specifying these parameters, you can disable automatic tiling parameter optimization, configure the tile size to 512x512, and set a 10% overlap between tiles.
+
+Run Tiling on OpenVINO Exported Model
+**************************************
+
+After training a model with tiling enabled, you can export the model to OpenVINO IR format using the following command:
+
+.. code-block:: 
+
+    otx export Custom_Object_Detection_Gen3_SSD --load-weights <path_to_trained_model>/weights.pth --output <path_to_exported_model>
+
+
+After exporting the model, you can run inference on the exported model using the following command:
+
+.. code-block:: 
+
+    ote eval Custom_Object_Detection_Gen3_SSD --test-data-roots tests/assets/small_objects --load-weights <path_to_exported_model>/openvino.xml
+
+.. warning::
+    Tiling trades off speed for accuracy as it increases the number of images to be processed. Therefore, it is normal to see a longer training and inference time when tiling is enabled.
```

Comment:
```suggestion
.. warning::
    
    Tiling trades off speed for accuracy as it increases the number of images to be processed. Therefore, it is normal to see a longer training and inference time when tiling is enabled.
```

Reply:
Also we can make another note that sometimes batchsize decrease is required to train on  machines that are not really powerful